packages/gatsby/src/bootstrap/index.js

```diff
@@ -80,7 +80,7 @@ module.exports = async (args: BootstrapArgs) => {
   if (config && config.__experimentalThemes) {
     const themesConfig = await Promise.mapSeries(
       config.__experimentalThemes,
-      async ([themeName, themeConfig]) => {
+      async ({ resolve: themeName, options: themeConfig }) => {
```

Comment:
Why don't we just resolve this in a "hacky" way (for now!) and do something like this?

```js
if (config && config.__experimentalThemes) {
  const themesConfig = await Promise.mapSeries(
    config.__experimentalThemes,
    async plugin => {
      const pluginName = plugin.resolve || plugin
      const themeConfig = plugin.options || {}
    }
  )
}
```

then we can totally avoid the issue of API idiosyncrasies, right?

Reply:
I updated the code to have this effect